charts/tidb-cluster/values.yaml

```diff
@@ -108,7 +108,7 @@ pd:
     # annotations: {} "<the default name will be used>"
     # portName: "client"
   replicas: 3
-  image: pingcap/pd:v3.0.13
+  image: pingcap/pd:v4.0.0
```

Comment:
The helm chart for tidbcluster is no longer maintained after operator 1.1 released. I'm not sure whether we should update it.